utils/environment.python.sh

```diff
@@ -2,9 +2,7 @@
 
 BASE_DIR=${FPGA_TOOL_PERF_BASE_DIR}
 # FIXME: fasm should be installed into the running Python environment.
-# FIXME: enable SYMBIFLOW when installed toolchain is available
-#export PYTHONPATH="${SYMBIFLOW}/python:$BASE_DIR:$BASE_DIR/third_party/fasm:$BASE_DIR/third_party/edalize:$PYTHONPATH"
-export PYTHONPATH="$BASE_DIR:$BASE_DIR/third_party/fasm:$BASE_DIR/third_party/edalize:$PYTHONPATH"
+export PYTHONPATH="${SYMBIFLOW}/bin/python:$PYTHONPATH"
```

Comment:
This seems wrong and probably it doesn't needs to be set anymore?

Does this script do a `conda activate` somewhere?

Reply:
Done, the script is actually not needed anymore as all PYTHON dependencies are already included.